collections/_blogs/2020-07-16-community-bridge-with-layer5.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Community Bridge 2020"
+date:   2020-07-16 10:30:05 -0530
+image: /assets/images/posts/2020-07-16-community-bridge-with-layer5/communitybridge.jpg
+author: Kanishkar J
+tags: community, open source, community bridge
+permalink: /blog/community-bridge-with-layer5
+---
+<img class = "thumbnail" src = "/assets/images/posts/2020-07-16-community-bridge-with-layer5/cblogo.png">
+<br><br>
+
+My journey into service meshes began on April 27, 2020 when I was selected to be a mentee under the banner of Community Bridge for the Summer of 2020. The Community Bridge Mentorship is a structured remote learning opportunity for aspiring open source software developers. The projects they host are mentored by experienced open source project developers and maintainers.
+
+<h4> Stumbling into Service Mesh Interface (SMI) </h4>
+
+I began working on SMI Conformance testing under the guidance of Lee Calcote, Founder of Layer5, in February 2020. Since then, my path into the world of service meshes has brought me an avalanche of opportunities and has helped me grow by leaps and bounds as a developer.
+<br>
+The Service Mesh Interface (SMI) is a standard interface that encapsulates various common service mesh functionalities. It explains several Custom Resource Definitions (CRDs), their configuration variables, and their corresponding effect on the service mesh underneath in full detail. What makes SMI vital is that these CRDs are generic, and the specifications are implemented and actively followed up by various service meshes. Some of these service meshes have developed adapters that define SMI CRDs as a wrapper over the APIs they already provide, instead of adhering to the SMI spec internally. SMI specifications address essential features for the most common service mesh use cases:
+<br>
+
+- Traffic policy – To apply policies like identity and transport encryption across services
+- Traffic telemetry – To capture key metrics like error rate and latency between services
+- Traffic management – To shift traffic between different services
+
+However, at the moment, no service meshes adhere to the SMI specifications completely. Service meshes are rapidly developing and making active efforts to extend their feature set. Simultaneously, SMI has also been working towards adding more features to its specifications.
+<br>
+Consequently, it becomes a necessity for continual validation of service mesh conformance with respect to SMI specifications and  the various versions of those specifications. The process of verifying conformance needs to be standardized and tooling made available.  Hence, our project efforts to create an SMI conformance tool.
+
+<h4> Meshery to the rescue </h4>
```

Comment:
Be consistent with capitalization in the headings. Some are all caps and some start with a capital letter and move to lowercase.